monai/transforms/utility/array.py

```diff
@@ -1015,3 +1016,25 @@ def _compute(op: Callable, data: np.ndarray):
                 raise ValueError("ops must be key string for predefined operations or callable function.")
 
         return img, meta_data
+
+
+class ToDevice:
```

Comment:
I think this should inherit Transform

Reply:
Thanks for pointing out the typo, updated in the latest commit.